### PR TITLE
Fix proj file getting updated even after XML editor closed

### PR DESCRIPTION
**Pull request by duncansmart**
The main reason I forked this was fix the above mentioned issue - this was addressed by b183dbf. Whilst I "had the bonnet open" I hope you don't mind that I made several other refactorings that I think  simplify the code.

### CODE REVIEW

**Comment by duncansmart** [comment]
Fixes #5

**Comment by dud5** [comment]
Hi Ducan. Thanks for the PR! I'll take a look at this weekend.

File: EditProj/EditProj.csproj [hunk 1]
```diff
@@ -47,6 +42,11 @@
     <WarningLevel>4</WarningLevel>
     <RunCodeAnalysis>true</RunCodeAnalysis>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <StartAction>Program</StartAction>
+    <StartProgram>C:\Program Files (x86)\Microsoft Visual Studio $(VisualStudioVersion)\Common7\IDE\devenv.exe</StartProgram>
```
Comment (dud5) [inline EditProj/EditProj.csproj]: Can the path
C:\Program Files (x86)
be obtained through a variable so as to not hardcode the drive letter?

**Comment by dud5** [comment]
Your change looks good to me.

Can you

1) Squash the commits into a "Clean up" and one fix for the bug
2) Address the minor comment about the path to program files

Thanks again!

**Comment by duncansmart** [comment]
I've slimmed down the original PR into one that just fixes #5 - I shall submit the others in another PR.

**Pull request merged**